tests/modules/nf-core/spaceranger/count/test.yml

```diff
@@ -0,0 +1,125 @@
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/raw_feature_bc_matrix.h5
+      md5sum: 2263d2c756785f86dc28f6b76fd61b73
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/raw_feature_bc_matrix/barcodes.tsv.gz
+      md5sum: 0508b5422699634fecc4ec04b44278c9
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/raw_feature_bc_matrix/features.tsv.gz
+      md5sum: feb8c7f35ee19b9bcd95d31511736c35
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/raw_feature_bc_matrix/matrix.mtx.gz
+      md5sum: 9421e93cb24280534d9c01e8c8eb5b75
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/raw_probe_bc_matrix.h5
+      md5sum: 3d5e711d0891ca2caaf301a2c1fbda91
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/aligned_fiducials.jpg
+      md5sum: 51dcc3a32d3d5ca4704f664c8ede81ef
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/cytassist_image.tiff
+      md5sum: 0fb04a55e5658f4d158d986a334b034d
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/detected_tissue_image.jpg
+      md5sum: 64d9adb4844ab91506131476d93b28dc
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/scalefactors_json.json
+      md5sum: 6180be5117729702e263e16dcd51ef49
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/tissue_hires_image.png
+      md5sum: 1c0f1e94522a886c19f56a629227e097
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/tissue_lowres_image.png
+      md5sum: 8c1fcb378f7f886301f49ffc4f84360a
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/spatial/tissue_positions.csv
+      md5sum: 1b2df34f9e762e9e64aa226226b96c4b
+    - path: output/spaceranger/CytAssist_11mm_FFPE_Human_Glioblastoma_2/outs/web_summary.html
+      contains:
+        - "CytAssist_11mm_FFPE_Human_Glioblastoma_2"
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/filtered_feature_bc_matrix.h5
+      md5sum: f444a4816bf40d377271a6157b320556
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/filtered_feature_bc_matrix/barcodes.tsv.gz
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/filtered_feature_bc_matrix/features.tsv.gz
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/filtered_feature_bc_matrix/matrix.mtx.gz
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/metrics_summary.csv
+      md5sum: 5e36f2f9b6987791e0b5eb2736d25115
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/molecule_info.h5
+      md5sum: 2669d8f1065d1314122aa129268f4c2f
+    - path: output/spaceranger/Visium_FFPE_Human_Ovarian_Cancer/outs/possorted_genome_bam.bam
+      md5sum: 6ed7f3bb2f17322113f940989a3771ff
```

Comment:
oooh how did you get stable MD5 sums for BAM, BAM indices, gzipped files, etc.? for Cellranger module tests they are typically unstable (maybe Cellranger doesn't sort them?)

Reply:
on my local machine, they were consistent between two runs. Not sure if this holds on CI, but it didn't run there...